src/test-helpers/factories.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+/**
+ * This service collects factories that create the entities defined in
+ * `types.js`, for use in writing automated tests.
+ */
+import path from 'path';
+import uuid from 'uuid/v1';
+
+const sept14thTimestamp = 1536927197731;
+
+export const createTask = (overrides: any = {}) => ({
```

Comment:
Is there a way to type this so it’s still type-safe? TypeScript has `Optional<T>` which allows you to pass as many keys as you want while still requiring any passed keys be of the correct type.

Reply:
Yeah, I can type it as `{[key: string]: any}` so that it's a map-like object. I'll update that